index.html

```diff
@@ -1212,9 +1221,18 @@
       <a>NDEF record</a> types.
     </p>
     <p>
-      The <dfn>NDEFRecord.data()</dfn> method returns the
-      payload data of the <a>NDEF record</a> with an appropriate [[!ECMAScript]]
-      type, which depends on the <a>MIME type</a>.
+      The <dfn>NDEFRecord.toText()</dfn> method, when invoked, MUST return the
+      result of running <a>convert an NDEFRecord object's bytes</a> with <i>text</i>,
+      unless there's other specific illustration in step 7 of
+      <a href="#the-nfc-reading-algorithm">The NFC reading algorithm</a>.
+    </p>
+    <p>
+      The <dfn>NDEFRecord.toArrayBuffer()</dfn> method, when invoked, MUST return the
+      result of running <a>convert an NDEFRecord object's bytes</a> with <i>arrayBuffer</i>.
+    </p>
+    <p>
+      The <dfn>NDEFRecord.toJSON()</dfn> method, when invoked, MUST return the
+      result of running <a>convert an NDEFRecord object's bytes</a> with <i>JSON</i>.
```

Comment:
`<i>JSON</I>`... italics doesn't convey any semantics? Avoid using that.

Reply:
It presents the parameter passed to the `<a>convert an NDEFRecord object's bytes</a>` algorithm. So we don't need the italics?

Reply:
See my other comment... that should be a `<var>` or something else? Definitely no use case for `<I>` so something is not right here.

Reply:
I will use the back-tick. :)